orm/iterator.go

```diff
@@ -92,6 +95,91 @@ func First(it Iterator, dest Persistent) (RowID, error) {
 	return binKey, nil
 }
 
+func Paginate(
+	it Iterator,
```

Comment:
Here we provide an `Iterator` because it needs to be created based on the query needs, see following examples:
- If we wanna query all items from an `orm` table, we can use `table.PrefixScan(ctx, start, end)` with `start = pageReq.Key` if `pageReq.Key` is provided to get an `Iterator`
- If we wanna query certain items based on certain conditions (eg query groups by admin), we can use corresponding `MultiKeyIndex`, eg `groupByAdminIndex.Get(ctx, searchKey)` where `searchKey` corresponds to the admin account address bytes. In this case, we don't really use `pageReq.Key` if provided and the returned `*query.PageResponse`.`NextKey` will return the next `RowID`, which doesn't seem to be usable in this particular context. But maybe I'm missing something here, any thoughts @aaronc?

Reply:
Can you explain why `pageReq.Key` isn't used in the second case?

Reply:
@aaronc nevermind I believe I figured it out, ie how to query a `MultiKeyIndex` (e.g. with the example of the `groupByAdminIndex`, querying groups by admin account address) only from a certain key, will upgrade the current implementation accordingly.